spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/filters/RateLimitPostFilter.java

```diff
@@ -78,8 +76,8 @@ public Object run() {
 
         policy(route, request).forEach(policy -> {
             long requestTime = System.currentTimeMillis() - getRequestStartTime();
-            String key = rateLimitKeyGenerator.key(request, route, policy);
-            rateLimiter.consume(policy, key, requestTime > 0 ? requestTime : 1);
+            String key = this.rateLimitKeyGenerator.key(request, route, policy);
+            this.rateLimiter.consume(policy, key, requestTime > 0 ? requestTime : 1);
         });
 
         return null;
```

Comment:
This class has nothing to do with the proposed changes, I think it can be reverted to the previous state

Reply:
I noticed that you had added the `this` keyword in quite a few places, I'm still going to add the coding checkstyle, but in this project, it's preferred to invoke the variable or method directly without the `this` keyword unless there's a conflicting class member and method member.

Reply:
ok, I think I have enabled this option in my ide. I will revert this changes.